Feature/1 project setup

jieun-dev1 opened the pull request:
Spring initializer 로 src, gradle 등 구성하였습니다. 
idea 등이 push 되지 않도록 git ignore 파일을 업데이트 하였습니다.

jieun-dev1 committed:
initial project setting

jieun-dev1 committed:
Chore: git ignore 업데이트

jieun-dev1 committed:
removed .idea files

f-lab-bright commented on .gitignore:
```diff
@@ -21,3 +21,41 @@
+HELP.md
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
```
이 파일은 같이 올라와야 추후에 다른 곳에서 빌드할 때 gradle 버전 의존성 없이 항상 같은 버전으로 빌드할 수 있습니다.
ignore 시켜두시더라도 한 번은 올려두셔야 해요 :)

jieun-dev1 replied:
gradle/wrapper/gradle-wrapper.jar 다시 커밋했습니다 :)

f-lab-bright reviewed (commented):
indent가 지금 전부 tab으로 되어있는데, 정리해서 space로 맞춰주시는게 좋을 것 같습니다.
lint는 나중에 다시 달더라두요 :)

jieun-dev1 committed:
Chore: 기존 파일의 tab을 space로 변경

convert to spaces 설정하여 기존 tap -> spaces 로 변경.
java Use tap character 해제로 tap이 space 로 인식 되게 함.

gitignore: 아래 파일 추가 - 다른 곳에서 빌드할 때, gradle 의존성 없이 같은 버전으로 빌드할 수 있게 하기 위함

.gradle
build/
!gradle/wrapper/gradle-wrapper.jar

f-lab-bright reviewed (dismissed):
LGTM with Comments
gradle-wrapper.jar 파일 여전히 안올라와있습니당.
요거 force add 해서 올려서 병합하시고 다음 작업 하나씩 하시면 됩니다 :)

f-lab-bright reviewed (commented):
어우 다 지워버리셨네요..?ㅋㅋ

jieun-dev1 committed:
gradle-wrapper.jar 추가

f-lab-bright reviewed (approved):
LGTM
